dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1625,4 +1626,133 @@ def put_student_in_section(student, teacher, script)
   test_user_gets_response_for :show, response: :success, user: :levelbuilder,
     params: -> {script_level_params(@pilot_script_level)},
     name: 'levelbuilder can view pilot script level'
+
+  def create_visible_after_script
```

Comment:
a better name would be `create_visible_after_script_level`, because it returns a script level, here and in other files